docs/source/matplotlib/advanced_plotting.rst

```diff
@@ -54,6 +54,7 @@ Block plots
     :include-source:
 
     import iris
+    import iris_sample_data
```

Comment:
That's an unused import... 😢

Reply:
(I know the *data* is used, but it is a wart to import something so that something else can do a thing that I'm really not keen on)